src/components/fair/booth.tsx

```diff
@@ -0,0 +1,179 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
+
+import Artworks from "../artwork_grid"
+import Button from "../buttons/ghost"
+import FollowButton from "../follow"
+import Text from "../text"
+
+import * as fonts from "../../assets/fonts"
+
+const PageSize = 8
+
+interface Props extends RelayProps, React.HTMLProps<FairBooth> {
+  show: any
```

Comment:
Out of curiosity, what does this do? Does TS use the definition from `RelayProps` because that one is more restricted than this `any` or does this `any` override the definition from `RelayProps`?